lib/gui/models/drives.js

```diff
@@ -20,103 +20,94 @@
  * @module Etcher.Models.Drives
```

Comment:
We can get rid of the JSDoc annotation. Sadly none of the models can be moved to shared yet given that the implicitly rely on localStorage (because of the redux store)